app/assets/stylesheets/views/_taxons.scss

```diff
@@ -23,13 +21,20 @@
 
   h3 {
     @include bold-19;
-    padding-bottom: $gutter-one-third;
+    @include media (tablet) {
+      padding-bottom: $gutter-one-third;
+    }
   }
 
   p {
     @include core-19;
-    margin-top: 2px;
-    margin-bottom: 6px;
+    margin-top: #{(5 / 16)}em;
+    margin-bottom: #{(20 / 16)}em;
+
+    @include media(tablet) {
+      margin-top: #{(5 / 19)}em;
+      margin-bottom: #{(20 / 19)}em;
```

Comment:
What's the significance of these values, and could it be made clearer with variables or comments? I might be missing some context, but it's not obvious to me why 20/19 em would be a useful measurement.